src/base/net/downloadmanager.cpp

```diff
@@ -2,6 +2,7 @@
  * Bittorrent Client using Qt and libtorrent.
  * Copyright (C) 2015-2024  Vladimir Golovnev <glassez@yandex.ru>
  * Copyright (C) 2006  Christophe Dumez <chris@qbittorrent.org>
+ * Copyright (C) 2024  Jonathan Ketchker <15849761+jNullj@users.noreply.github.com>
```

Comment:
1. As you might have noticed, we are adding new copyrights to the top.
2. If you don't want to provide your real email, just don't do it, you don't need to use such "noreply" address.

Reply:
Fixed at 8f6218e